test/functorch/test_aotdispatch.py

```diff
@@ -429,13 +428,46 @@ def verify_aot_autograd(
                     if test_mutation:
                         # This tests that autograd meta is set properly on the output we can
                         # mutate it.
-                        ref_o.mul_(2)
-                        test_o.mul_(2)
+                        ref_o.add(2)
```

Comment:
we probably want to keep the `mul_(2)` from the line above, instead of doing an out-of-place add. The test is trying to confirm that we can mutate the output of a compiled region without making autograd unhappy / raising an error

Reply:
Hmm ok, I chose add because it was reversible but I can probably just recreate a new ref_out each time

Reply:
oh `add_` is fine too - what I meant is that the test is trying to inplace-mutate the data of the output of the compiled function, to ensure that autograd won't error on the mutation (so we lose test coverage if we chaneg it to be an out-of-place op like `test_o.add()`)

Reply:
Oh I did not realize .`add()` was out of place instead of `add_`. Will change!